### PR TITLE
Fix detection of word boundaries to not include hyphan as boundary

### PR DESCRIPTION
**Pull request by HonzaMac**
Adding failing tests for issue https://github.com/btford/write-good/issues/112

### CODE REVIEW

File: test/basic.spec.js [hunk 1]
```diff
@@ -29,6 +30,18 @@ describe('too-wordy', () => {
     });
   });
 
+  describe('a sentence with hyphen should parse word as one', () => {
```
Comment (RichardLitt) [inline test/basic.spec.js]: Should be rephrased: "hyphenated words should not be parsed as two words". I think this is more clear.

**Review by RichardLitt — CHANGES_REQUESTED** [review]

**Comment by RichardLitt** [comment]
Thanks, @HonzaMac. LGTM.

**Comment by RichardLitt** [comment]
@duereg Any reason this wasn't merged?

**Comment by HonzaMac** [comment]
There was only failing test, but no actual fix. 
There was no action for more then 1 year.

**Comment by HonzaMac** [comment]
@RichardLitt I've made fix, please review.

**Comment by RichardLitt** [comment]
My review isn't worth much. I'm not the maintainer.

**Comment by HonzaMac** [comment]
@duereg Please, can you take a look?

**Comment by duereg** [comment]
bumped to 0.3.0. Thanks!

**Comment by HonzaMac** [comment]
Thank you!

**Pull request merged**